keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaTaskLauncher.kt

```diff
@@ -63,6 +63,15 @@ class OrcaTaskLauncher(
       type = SubjectType.RESOURCE
     )
 
+  override fun submitJobAsync(
```

Comment:
@robfletcher This was the best I could come up with for a Java-friendly version of the method. It relies on `kotlin-coroutines-jdk8`, which I'm not sure how that fits with our plans to move to Java 11. Also, I found indications that the use of `GlobalScope` is discouraged, but wasn't sure what the alternative could be in this case.